toolparameteriser/testcreator.py

```diff
@@ -6,25 +6,45 @@
 from string import Template
 import xml.etree.ElementTree as ET
 import toolparameteriser.utils
-toolparameteriser.utils.setlogging()
+import subprocess
+# 
 
 class AbstractTester(ABC):
     def __init__(self,config:dict) -> None:
         super().__init__()
         self.tmplfile="" #Must be set by concrete class
         self.Config=config
-        
-        self.jobs_completed_file=os.path.join(self.Config['output']['path'],"jobs_completed.csv")
+
+        toolparameteriser.utils.setlogging(self.Config["debug"])
+
+        # Creating output directory
+        logging.debug("Checking if output path, {p}, exists.".format(p=self.Config['output']['path']))
```

Comment:
f-strings are the future! (same can be applied to all your `.format()` calls)
```suggestion
        logging.debug(f"Checking if output path, {self.Config['output']['path']}, exists.")
```

Reply:
Also, I would assign `self.Config['output']['path']` to a temporary variable here so to avoid repeating yourself.